ft_client/freqtrade_client/ft_rest_client.py

```diff
@@ -269,6 +269,36 @@ def trades(self, limit=None, offset=None):
             params["offset"] = offset
         return self._get("trades", params)
 
+    def list_open_trades_custom_data(self, key=None, limit=100, offset=0):
+        """List open trades custom-data of the running bot.
+
+        :param key: str, optional - Key of the custom-data
+        :param limit: limit of trades
+        :param offset: trades offset for pagination
+        :return: json object
+        """
+        params = {}
+        params["limit"] = limit
+        params["offset"] = offset
+        if key is not None:
+            params["key"] = key
+
+        return self._get("trades/open/custom-data", params=params)
```

Comment:
`freqtrade-client --config partial_api.json list_custom_data 5` returns the following:

`{"detail": [{"type": "int_parsing", "loc": ["path", "trade_id"], "msg": "Input should be a valid integer, unable to parse string as an integer", "input": "{tradeid}"}]}`

Reply:
Yes I'm on it, I will let you know when it is fixed